### ISSUE TITLE
Output HTML table is narrow on some browsers

### ISSUE THREAD

**Issue by tyronerees** [fitbenchmarking/fitbenchmarking#685]
This is the output of the HTML tables when fitbenchmarking is run using Mozilla Firefox 62.0.3 (Fedora), opened through an ssh -X session. 

![Screenshot from 2020-11-06 14-28-39](https://user-images.githubusercontent.com/13645545/100867605-57552700-3492-11eb-9ec5-7c0ee566ea8a.png)

Display is ok in Firefox 82.0.

The same issue appears when displayed in seamonkey run via WSL.

**Comment by AndrewLister-STFC** [comment]
I've pinned down the version to [firefox 66.0](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/66) as the one where this was fixed.
It looks like setting the max-width to max-content only works after this version.
```
div.container {
    max-width: max-content;
    width: 80%;
    ...
```

This can be fixed in earlier versions by prefixing max-content with -moz-.
```
div.container {
    max-width: -moz-max-content;
    width: 80%;
    ...
```

Should we still try to support this? It was released in March 2019, but updating a browser shouldn't be a sticking point for anyone?

I'd suggest this could be fixed with documentation
